<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterTableWidgetForm</class>
 <widget class="QWidget" name="ParameterTableWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QTableWidget" name="m_paramTableWidget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Parameter</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="m_addItemToolButton">
         <property name="toolTip">
          <string>Add a new item</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_removeItemToolButton">
         <property name="toolTip">
          <string>Remove selected item</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_editItemToolButton">
         <property name="toolTip">
          <string>Edit selected item</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_paramTableWidget</tabstop>
  <tabstop>m_addItemToolButton</tabstop>
  <tabstop>m_removeItemToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
